raiden-ts/src/config.ts

```diff
@@ -37,8 +37,9 @@ const RTCIceServer = t.type({ urls: t.union([t.string, t.array(t.string)]) });
  *    use null to disable
  * - pfs - Path Finding Service URL or Address. Set to null to disable, or empty string to enable
  *    automatic fetching from ServiceRegistry.
- * - pfsSafetyMargin - Safety margin to be added to fees received from PFS. Use `1.1` to add a 10%
- *    safety margin.
+ * - pfsSafetyMargin - Safety margin to be added to fees received from PFS. Either a fee
+ *    multiplier, or a [fee, amount] pair ofmultipliers. Use `1.1` to add a 10% over estimated fee
+ *    margin, or `[0.03, 0.0005]` to add a 3% over fee plus 0.05% over amount.
```

Comment:
One of our objectives is compliance with the reference implementation. This was neglected so far because we didn't have this feature in place, but now that we're adding it, we need to comply, and these parameters are required to calculate the safety margin the same the PC does.

Reply:
Split out as well.